drivers/ieee802154/ieee802154_uart_pipe.c

```diff
@@ -404,10 +400,10 @@ static inline uint8_t *get_mac(const struct device *dev)
 	UNALIGNED_PUT(sys_cpu_to_be32(sys_rand32_get()),
 		      (uint32_t *) ((uint8_t *)upipe->mac_addr+4));
 #else
-	upipe->mac_addr[4] = CONFIG_IEEE802154_UPIPE_MAC4;
-	upipe->mac_addr[5] = CONFIG_IEEE802154_UPIPE_MAC5;
-	upipe->mac_addr[6] = CONFIG_IEEE802154_UPIPE_MAC6;
-	upipe->mac_addr[7] = CONFIG_IEEE802154_UPIPE_MAC7;
+	upipe->mac_addr[4] = CONFIG_IEEE802154_BM_UPIPE_MAC4;
+	upipe->mac_addr[5] = CONFIG_IEEE802154_BM_UPIPE_MAC5;
+	upipe->mac_addr[6] = CONFIG_IEEE802154_BM_UPIPE_MAC6;
+	upipe->mac_addr[7] = CONFIG_IEEE802154_BM_UPIPE_MAC7;
 #endif
```

Comment:
Are changes in this file required anymore given the new driver?

Reply:
Looks good as a starting point for the BM drivers.

One change that should be made is the inclusion of the LICENSE.txt file for cobs-c.